cv19ResSupportV3/V3/Gateways/HelpRequestGateway.cs

```diff
@@ -60,6 +60,36 @@ public int CreateHelpRequest(int residentId, CreateHelpRequest command)
                 throw;
             }
         }
+        public int? FindHelpRequestByMetadata(string propertyName, dynamic metadata)
+        {
+            string propertyInfo;
+
+            try
+            {
+                propertyInfo = metadata.GetProperty(propertyName).ToString();
+            }
+            catch (Exception e)
+            {
+                return null;
+            };
+
+            var sqlString = $"select * from help_requests WHERE metadata->>'{propertyName}' LIKE '{propertyInfo}'";
```

Comment:
:vomiting_face:  that JSON accessor.


@elena-vi can you use prepared statements here please https://docs.microsoft.com/en-us/ef/core/querying/raw-sql#passing-parameters